lib/internal/test_runner/runner.js

```diff
@@ -130,7 +146,12 @@ function getRunArgs(path, { forceExit, inspectPort, testNamePatterns, testSkipPa
   if (only === true) {
     ArrayPrototypePush(argv, '--test-only');
   }
-  ArrayPrototypePush(argv, path);
+
+  if (path === kIsolatedProcessName) {
+    ArrayPrototypePush(argv, '--test', ...ArrayPrototypeSlice(process.argv, 1));
```

Comment:
<https://github.com/nodejs/node/blob/main/doc/contributing/primordials.md#unsafe-array-iteration> "Avoid spread operator on arrays"?

Does this fall under that?

Reply:
No. This is an argument to a function